_lessons/17-adding-placeholder-text.md

```diff
@@ -0,0 +1,196 @@
+
+The `text-align` property can have the values left, right, or center. Let's go ahead and add `text-align: center;` to the header and footer: 
+
+```
+#header {
+  width: 800px;
+  margin-left: auto;
+  margin-right: auto; 
+  text-align: center;
+}
+#footer {
+  width: 800px;
+  margin-left: auto;
+  margin-right: auto; 
+  text-align: center;
+}
+```
+
+Now the header and footer are centered:
+
+<div class="screenshot">
+    <img src="/assets/ch17_4.png" />
+</div>
+
+### Making Links Full-Width
+
+The last problem with our page is that our links are all on the same line, wrapping, in the sidebar. We want each link to take up its own line. 
+
+The reason the links are wrapping has to do with the CSS `display` property. This property controls how the element flows in the document. This property has several values, but we'll focus on two right now: 
+
+- `inline` means that the element flows in line with the text. Texts and links are inline. These elements **don't have a fixed width** - think about it, when you type a bit of text, it just takes up what it takes up. If you write multiple inline elements, they will flow in a row one after the other. 
+- `block` means that the element gets its own section of the page. For example, a chart or figure in a document would show up on its own row. If you write multiple block elements, they will each sit on their own row, as if there are line breaks in between them. 
+
+By default, links (the `a` tag) are inline because they are often found in the middle of text. We can make them behave like block elements by setting `display: block;` on the `a` tag: 
+
+```
+a {
+  display: block; 
+}
+```
+
+Now, our links will each take up their own row. 
+
+<div class="screenshot">
+    <img src="/assets/ch17_5.png" />
+</div>
+
+### Selecting Only Sidebar Links
+
+There is one problem with the code we just added: since it will apply to every `a` tag on the page, it will apply to links in the content as well, not just the sidebar! Let's try adding a link to the content just to see: 
+
+```
+<div id="content">
+  <p>Here is a <a href="#">link inside of my content text</a> to see what happens.</p>
+  <p>Lorem ipsum dolor sit amet, consectetur adipiscin gelit. [...and so on]
+</div>
+```
+
+Uh oh - this doesn't look right. The link is showing up on its own line, instead of in line with the text.
+
+<div class="screenshot">
+  <img src="/assets/ch17_6.png" />
+</div>
+
+We need a way to apply our CSS to *just* the `a` tags in the sidebar, not on the entire page. It's time to introduce the word *CSS selector.*
+
+A CSS selector is the part of the CSS that we've been writing *before* the curly braces, to tell the CSS what elements to apply our rules to. 
+
+```
+SELECTOR {
+  PROPERTY: VALUE;
+}
+```
```

Comment:
I'm not sure this is the right place to be introducing fundamental terminology. The introduction of the word "**selector**" should probably go along with the introductions of the terms "**property**" and "**value**" in Lesson 8, since all three of those words are IMO of the same level of importance.